frontend/src/models/backend.ts

```diff
@@ -32,6 +32,8 @@ type IBackendState = ILookup<any> & {
   }
   preferences: IPreferences
   deferredAttributes?: IService['attributes']
+  reachablePort: boolean
+  loading: boolean
```

Comment:
Maybe this var should be renamed `reachablePortLoading` so that it's specific instead of general could cause confusion. Thanks!